tools/bwa.wdl

```diff
@@ -66,69 +66,69 @@ task bwa_aln {
 
     runtime {
         memory: memory_gb + " GB"
-        disk: disk_size + " GB"
+        disk: disk_size_gb + " GB"
         cpu: ncpu
         docker: 'ghcr.io/stjudecloud/bwa:0.7.17-0'
         maxRetries: max_retries
     }
 }
 
 task bwa_aln_pe {
-
     meta {
-        author: "Andrew Thrasher"
-        email: "andrew.thrasher@stjude.org"
         description: "This WDL task maps paired-end FastQ files to BAM format using bwa aln."
     }
 
     parameter_meta {
-        fastq1: "Input FastQ read 1 file to align with bwa"
-        fastq2: "Input FastQ read 2 file to align with bwa"
+        read_one_fastq_gz: "Input FastQ read 1 file to align with bwa"
+        read_two_fastq_gz: "Input FastQ read 2 file to align with bwa"
         bwadb_tar_gz: "Gzipped tar archive of the bwa reference files. Files should be at the root of the archive."
     }
 
     input {
-        File fastq1
-        File fastq2
-        String output_bam = basename(fastq1, ".fq.gz") + ".bam"
+        File read_one_fastq_gz
+        File read_two_fastq_gz
         File bwadb_tar_gz
+        String prefix = basename(read_one_fastq_gz, ".fq.gz")  # TODO is this right?
```

Comment:
@adthrasher everywhere else we use a `.fastq` extension. Do you have any idea whether `.fastq` or `.fq` is more common? Should we try and handle both? It's ugly but the below snippet should work:
```
String prefix = basename(basename(read_one_fastq_gz, ".fq.gz"), ".fastq.gz")
```